src/components/app.jsx

```diff
@@ -47,6 +47,12 @@ const App = React.createClass({
               </div>
             </section>
             {this.props.children}
+            <footer style={{marginTop: "5em"}}>
```

Comment:
I added a stylesheet.

I agree for performance and UI unity (use the same patterns everywhere).
As a side note, about maintainability, I think that putting styles in separate `.css` files is more a separation of technologies, than a separation of concerns. To develop further we could discuss about it IRL not to pollute the PR ;-)